Description: Adds a p after assertRaisesRegex()
 The new version isn't in Debian, so we fix it.
Author: Thomas Goirand <zigo@debian.org>
Forwarded: not-needed
Last-Update: 2015-04-17

--- cinder-2015.1~rc1.orig/cinder/tests/objects/test_objects.py
+++ cinder-2015.1~rc1/cinder/tests/objects/test_objects.py
@@ -438,7 +438,7 @@ class _TestObject(object):
         class Foo(base.CinderObject):
             fields = {'foobar': fields.Field(fields.Integer())}
         obj = Foo()
-        with self.assertRaisesRegex(NotImplementedError, ".*foobar.*"):
+        with self.assertRaisesRegexp(NotImplementedError, ".*foobar.*"):
             obj.foobar
 
     def test_loaded_in_primitive(self):
